manifests/params.pp

```diff
@@ -217,4 +217,9 @@
     'TrackSlurmctldDown' => 'no',
     'TrackWCKey' => undef,
   }
+
+  $env_dir = $::$::osfamily ? {
```

Comment:
per comments in #22 this should be in `init.pp` as a parameter and then change the default value using Hiera in `data/os`.